src/lib/libast/path/pathtemp.c

```diff
@@ -113,6 +113,18 @@ static struct Tmp_s {
     char *tmppath;
 } tmp = {.mode = S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH};
 
+static_fn char *get_tmp_dir() {
```

Comment:
This name might clash with other names in debug builds in future. May be rename it to `get_pathtmp_dir`.